app/src/main/res/layout-sw320dp/activity_upload_activity.xml

```diff
@@ -26,161 +25,157 @@
             android:id="@+id/uploadActivityTitle"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:text="New Activity"
+            android:text="@string/activity_upload_uploadActivityTitle"
```

Comment:
Why are you not following the same convention as before (the one where you write everything in lowercase with underscores) from here?

Reply:
good observation ! It's a mistake,I will change that ! thanks

Reply:
Ok take care, because this was not only in this file, some other files miss your initial convention as well.